Dockerfile

```diff
@@ -1,6 +1,5 @@
-# Dockerfile
 # Base image with Python 3.11 on Alpine
-FROM python:3.11-alpine
+FROM python:3.11
```

Comment:
why this change?

Reply:
this was fixed by @k-allagbe I just squashed his commit

Reply:
`ailab-db`, which this app depends on, requires tools that are not present in the `alpine` variant for it's dependencies installation. The tools could be installed on top of `alpine` via the dockerfile but they take very long to build, which could be painful in github actions. I bumped the variant all the way up to avoid all that.

Reply:
Done.